### PR TITLE
Competition by month

### PR DESCRIPTION
**Pull request by nmking22**
* Change competition query argument from competition ID to month
* Adapt tests to match change

## Types of Changes

This PR closes #

- [ ] Bug fix (non-breaking change which fixes an issue): 

- [ ] New feature (non-breaking change which adds functionality):

- [ ] Breaking change (fix or feature that would cause existing functionality to change):

- [ ] My change requires a change to the documentation.

- [ ] I have updated the documentation accordingly.

- [ ] I have added tests to cover my changes.

- [x] All new and existing tests passed.

### CODE REVIEW

**Review by the-color-bliu — APPROVED** [review]

File: app/graphql/types/query_type.rb [hunk 1]
```diff
@@ -11,19 +11,19 @@ class QueryType < Types::BaseObject
...
-    
+
     def competitions
       Competition.all
     end
 
     field :competition, Types::CompetitionType, null: false do
-      argument :id, ID, required: true
+      argument :month, Int, required: true
     end
```
Comment (ETBassist) [inline app/graphql/types/query_type.rb:23]: Dope, looks good; we might also want to add an argument for year, but this'll definitely do the thing for now

**Review by ETBassist — APPROVED** [review]
Yep

**Review by SageOfCode — APPROVED** [review]
Great work here.

File: app/graphql/types/query_type.rb (hunk 1 above)
Comment (nmking22) [reply app/graphql/types/query_type.rb:23]: Good call. I'll open up an issue for this

**Pull request merged**